openapi.yaml

```diff
@@ -0,0 +1,698 @@
+          description: name of game to return
+        - name: score_min
+          in: query
+          type: string
+          required: false
+          description: Filter results by minimum review score
+        - name: score_max
+          in: query
+          type: string
+          required: false
+          description: Filter results by maximum review score
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResults'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    post:
+      summary: Add a new game
+      tags:
+        - Games
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/GamePostBody'
+      responses:
+        '201':
+          description: Successfully added a game
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /games/{gameId}:
+    parameters:
+    - $ref: '#/parameters/authorization'
+    - name: gameId
+      in: path
+      description: Id of game to return
+      required: true
+      type: integer
+      format: int64
+    get:
+      summary: Find game by ID
+      description: Returns a single game
+      tags:
+        - Games
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      summary: Update information on a game
+      description: Pass in parameters to update fields on game resources
+      tags:
+        - Games
+      parameters:
+      - name: body
+        in: body
+        required: true
+        schema:
+          $ref: '#/definitions/GamePatchBody'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/GameResult'
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    delete:
+      summary: Delete a game
+      description: Delete a game by passing in a gameId
+      tags:
+        - Games
+      responses:
+        '200':
```

Comment:
Base on [JSON API](https://jsonapi.org/format/#crud-deleting), a successful DELETE should return a 202 (Accepted) or 204 (No Content).
Example: https://github.com/osu-mist/notes-api/blob/master/openapi.yaml#L182

Reply:
changed all successful deletes to 204